auth/authenticate.js

```diff
@@ -1,7 +1,8 @@
 const jwt = require('jsonwebtoken');
+const { jwtSecret } = require('../config/secret');
 
 const jwtKey =
-  process.env.JWT_SECRET ||
+  process.env.JWT_SECRET || jwtSecret
   'add a .env file to root of project with the JWT_SECRET variable';
```

Comment:
This probably ought to be accounted for (even if it doesn't produce an error); currently you have an `||` that confusingly defaults to `jwtSecret`, but there's a string attached on in the code: `'add a .env file to root of project with the JWT_SECRET variable'`.